test/e2e_node/memory_manager_test.go

```diff
@@ -621,7 +621,7 @@ var _ = SIGDescribe("Memory Manager [Disruptive] [Serial] [Feature:MemoryManager
 						return false
 					}
 
-					if !strings.Contains(tmpPod.Status.Message, "Pod Allocate failed due to [memorymanager]") {
+					if !strings.Contains(tmpPod.Status.Message, "Pod was rejected: Allocate failed due to [memorymanager]") {
```

Comment:
yes, it is unfortunate that our CI cannot run these tests yet :\
It's also unfortunate we make logic over the error message text, but I guess fixing this instance isn't trivial, being a e2e test.
Since we're tracking the error message, I'd rather trim down the text we look for rather than making it bigger. Something like
```
if !strings.Contains(tmpPod.Status.Message, "Allocate failed due to [memorymanager]") {
}
```
(provided we check for error/rejection right above) should be unambiguous and more future-proof. WDYT?

Reply:
That's a good suggestion, will update!